projects/aws/rolesanywhere-credential-helper/Makefile

```diff
@@ -5,8 +5,10 @@ REPO=rolesanywhere-credential-helper
 REPO_OWNER=aws
 
 BINARY_TARGET_FILES=aws_signing_helper
-SOURCE_PATTERNS=./cmd/aws_signing_helper
+SOURCE_PATTERNS=.
+EXTRA_GOBUILD_FLAGS=-buildmode=pie
 EXTRA_GO_LDFLAGS=-X k8s.io/component-base/version.gitVersion=$(GIT_TAG)
+CGO_CREATE_BINARIES=true
```

Comment:
i dont want to do this if its just for buildmode pie. that tends to be a sec thing which for static go binaries doesnt really seem neccessary.